tests/drivers/build_all/gpio/app.overlay

```diff
@@ -143,6 +143,15 @@
 					pinmux_mask = <0xff>;
 				};
 			};
+
+                        test_i2c_tca6424a: tca6424a@23 {
```

Comment:
This looks like it is missing a compatible.

Reply:
The compatible has been added

Reply:
... and you are mixing tabs and spaces. Please use tabs.

Reply:
The indents now consistently use tabs instead of spaces.